packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,516 @@
+placing selection of a Menu in an uncontrolled state. Alternatively, the
+`selectedKeys` prop preselects an Item in the Menu placing Item selection in a
+controlled state.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Edit (Controlled)
+  </ActionButton>
+  <Menu selectionMode="single" selectedKeys={['copy']}>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Edit (Uncontrolled)
+  </ActionButton>
+  <Menu selectionMode="single" defaultSelectedKeys={['paste']}>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+The `selectionMode` prop specifies how many Menu Items can be selected, with
+the options of a single Menu Item, multiple Menu Items, or disabling selection
+entirely (default).
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Show (Multiple)
+  </ActionButton>
+  <Menu selectionMode="multiple" defaultSelectedKeys={['Sidebar', 'Console']}>
+    <Item uniqueKey='Sidebar'>Sidebar</Item>
+    <Item uniqueKey='Searchbar'>Searchbar</Item>
+    <Item uniqueKey='Tools'>Tools</Item>
+    <Item uniqueKey='Console'>Console</Item>
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Edit (Selection Mode None)
+  </ActionButton>
+  <Menu selectionMode="none">
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+### Sections
+
+Menus may have Sections which can be used to wrap groups of Items. Each
+`Section` takes a `title` and `uniqueKey` prop.
+
+#### Static Items
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Edit
+  </ActionButton>
+  <Menu>
+    <Section uniqueKey="rollback" title="Rollback Options">
+      <Item uniqueKey="undo">Undo</Item>
+      <Item uniqueKey="redo">Redo</Item>
+    </Section>
+    <Section uniqueKey="select" title="Selected Text Options">
+      <Item uniqueKey="cut">Cut</Item>
+      <Item uniqueKey="copy">Copy</Item>
+      <Item uniqueKey="paste">Paste</Item>
+    </Section>
+  </Menu>
+</MenuTrigger>
+```
+
+#### Dynamic Items
+
+Sections should be populated with dynamic Items from a hierarchical data
+structure. Section takes an array of data using the `items` prop.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      File Types
+  </ActionButton>
+  <Menu
+    items={[{name: 'Docs', children: [{name: 'PDF'}]}, {name: 'Images', children: [{name: 'jpeg'}, {name: 'png'}, {name: 'tiff'}]}]}
+    itemKey="name">
+    {item => (
+      <Section items={item.children} title={item.name}>
+        {item => <Item>{item.name}</Item>}
+      </Section>
+    )}
+  </Menu>
+</MenuTrigger>
+```
+
+### Complex Content Elements
```

Comment:
Trying a new name instead of semantic.

Reply:
I think this name is fine, `Complex Menu Items` could work as well